src/components/AccessControlNew.tsx

```diff
@@ -127,6 +138,13 @@ const AccessControlNew = () => {
         })
     };
 
+    const handleChangeDisabled = ({ target: { value } }: RadioChangeEvent) => {
+        setFormRule({
+            ...formRule,
+            Disabled: value
```

Comment:
Codacy has a fix for the issue: [Replace `············Disabled:·value` with `······Disabled:·value,`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      Disabled: value,
```